assets/sass/imports/_presentations.scss

```diff
@@ -0,0 +1,55 @@
+.presentation {
+    position: relative; 
+    display: flex; 
+    flex-direction: column;
+    justify-content: start;  
+    margin-left:-40%;
+    margin-right:-40%;
+    padding: 3%;
+    @media (max-width: 1340px) {
+        margin-left:0%;
+        margin-right:0%;
+    }
+}
+
+.presentation .presentation_section {
+    display: flex; 
+    flex-direction: row; 
+    justify-content: space-between; 
+    width: 100%;
+    flex-flow: wrap;
+}
+
+.presentation .presentation_section .info-box {
+    width: 19%; 
+    display: flex;
+    justify-content: center; 
+    align-items:center; 
+    padding: 2%;
+    @media (max-width: 1340px) {
+        width: 100%;
+        order: 1;
+        padding-bottom: 0.6%;
+    }
+}
+
+.presentation .presentation_section .preview-box {
+    width: 70% ; 
+    padding: 1%;
+    @media (max-width: 1340px) {
+        width: 100%;
+        order: 2;
+        padding-top: 0%;
+        padding-bottom: 2%;
+    }
+}
+
+.presentation .presentation_section .preview-box .preview-pdf{
+    height: 500px; 
```

Comment:
On mobile devices there's lot of empty vertical space due to fixed height. Can you maybe make height aware of width (perhaps using aspect-ratio property may help)?

Reply:
Done in e6107cf